treeherder/seta/runnable_jobs.py

```diff
@@ -1,5 +1,7 @@
+'''Bug 1330649 - Get rid of this module'''
```

Comment:
I think this should be a standard comment rather than a docstring, and perhaps add the prefix TODO. eg:
`# TODO: Get rid of this module (bug 1330649).`